keps/sig-node/127-user-namespaces/README.md

```diff
@@ -363,88 +549,463 @@ Note this section is a WIP yet.
+https://git.k8s.io/enhancements/keps/sig-architecture/1194-prod-readiness.
+
+The production readiness review questionnaire must be completed and approved
+for the KEP to move to `implementable` status and be included in the release.
+
+In some cases, the questions below should also have answers in `kep.yaml`. This
+is to enable automation to verify the presence of the review, and to reduce review
+burden and latency.
+
+The KEP must have a approver from the
+[`prod-readiness-approvers`](http://git.k8s.io/enhancements/OWNERS_ALIASES)
+team. Please reach out on the
+[#prod-readiness](https://kubernetes.slack.com/archives/CPNHUMN74) channel if
+you need any help or guidance.
+-->
+
 ### Feature Enablement and Rollback
 
-* **How can this feature be enabled / disabled in a live cluster?**
-  - [ ] Feature gate
-    - Feature gate name:
-    - Components depending on the feature gate:
+<!--
+This section must be completed when targeting alpha to a release.
+-->
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+<!--
+Pick one of these and delete the rest.
+
+Documentation is available on [feature gate lifecycle] and expectations, as
+well as the [existing list] of feature gates.
 
-* **Does enabling the feature change any default behavior?**
+[feature gate lifecycle]: https://git.k8s.io/community/contributors/devel/sig-architecture/feature-gates.md
+[existing list]: https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/
+-->
+
+- [x] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: UserNamespacesSupport
+  - Components depending on the feature gate: kubelet, kube-apiserver
+
+###### Does enabling the feature change any default behavior?
+
+No.
+
+<!--
+Any change of default behavior may be surprising to users or break existing
+automations, so be extremely careful here.
+-->
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes. Pods that were created and opted-in to use user namespaces, will have to be
+recreated to run again without user namespaces.
+
+Other than that, it is safe to disable once it was enabled.
+
+<!--
+Describe the consequences on existing workloads (e.g., if this is a runtime
+feature, can it break the existing applications?).
 
-* **Can the feature be disabled once it has been enabled (i.e. can we roll back
-  the enablement)?**
+Feature gates are typically disabled by setting the flag to `false` and
+restarting the component. No other changes should be necessary to disable the
+feature.
 
-* **What happens if we reenable the feature if it was previously rolled back?**
+NOTE: Also set `disable-supported` to `true` or `false` in `kep.yaml`.
+-->
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+Pods will have to be re-created to use the feature.
+
+###### Are there any tests for feature enablement/disablement?
+
+We will add.
```

Comment:
> Or do you meant that it should be the same answer but s/Yes, /No, /, as there is not anything merged yet? If that is the case, sure, I can change that to "no". I just answered yes because we planned to merge the code and these tests, sorry if that caused confusion :)

That's what I meant sorry.
I want the KEP to reflect the current state and currently we obviously don't have any tests.
The current wording lgtm.

Reply:
Thanks!